webapi/src/main/scala/org/knora/webapi/responders/admin/RestProjectsService.scala

```diff
@@ -6,10 +6,23 @@ import zio.ZLayer
 import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetRequestADM
 import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetResponseADM
 import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsGetRequestADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsGetResponseADM
 import org.knora.webapi.responders.ActorToZioBridge
 
 final case class RestProjectsService(bridge: ActorToZioBridge) {
 
+  /**
+   * Returns all projects as a [[ProjectsGetResponseADM]].
+   *
+   * @return
+   *     '''success''': information about the projects as a [[ProjectsGetResponseADM]]
+   *
+   *     '''failure''': [[dsp.errors.NotFoundException]] when no project was found
```

Comment:
what is actually the deal with these triple-single-quotes? should they render in a certain way, or is it just convention? (just out of curiosity)

Reply:
I'm not sure, @seakayone you introduced it, could you explain what is behind these?
I just made it consistent, so that the same pattern is used everywhere.

Reply:
These triple-single-quotes render '''the text''' in bold, at least in my IDE.

Reply:
see also https://alvinalexander.com/scala/scaladoc-syntax-tags-wiki-formatting-examples/